py/tests/encoders/date_test.py

```diff
@@ -51,7 +52,7 @@ def testDateEncoder(self):
 
     # Week is MTWTFSS contrary to localtime documentation, Monday = 0 (for
     # python datetime.datetime.timetuple()
-    dayOfWeekExpected = [0,0,0,1,0,0,0]
+    dayOfWeekExpected = [0,0,0,0,1,0,0]
```

Comment:
We must review the correctness of the tests. The day should be Thrs, according to the comment above, the expected is now Fri, why? I think that was the reason I added the "normalization" to [0, 1/2].

Reply:
You've mentioned this is due to "small bitness" of the demo test, so that has to be mentioned in the comments. Preferably even changed so the encoding fits the logic that "Thursday afternoon" is still Thrs.